catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceProfilerPipeline.json

```diff
@@ -25,6 +25,13 @@
       "description": "Option to turn on/off generating sample data.",
       "type": "boolean",
       "default": true
+    },
+    "profileSample": {
```

Comment:
great! we'll need to reach out to the UI team so that they can add this property when configuring the Profiler Pipeline. Let's do those changes in this same PR 🙏